.github/workflows/trusted-cd-linux-release.yml

```diff
@@ -20,8 +20,8 @@ jobs:
           sudo apt-get install -y
           sudo apt-get remove -y llvm-12
           sudo apt-get install -y curl pkg-config findutils  wget
-          sudo apt install -y clang-11 cmake clang-format-11 clang-tidy-11
-          sudo apt-get install -y llvm-11 lldb-11 llvm-11-dev libllvm11 llvm-11-runtime
+          sudo apt install -y clang-13 cmake clang-format-13 clang-tidy-13
+          sudo apt-get install -y llvm-13 lldb-13 llvm-13-dev libllvm11 llvm-13-runtime
           sudo apt install -y python3 python3-pip
           sudo apt-get remove -y llvm-10
```

Comment:
(nit)
I assume there are [some specific reasons](https://www.google.com/search?q=what%27s+the+difference+between+apt+and+apt-get&rlz=1C1GCEU_enUS858US858&oq=what%27s+the+difference+between+apt+and+&aqs=chrome.2.0i512l3j69i57j0i22i30l6.10121j1j7&sourceid=chrome&ie=UTF-8) for using sometimes `apt` and sometimes `apt-get` in L20 - L26 (rather than either `apt` or `apt-get` everywhere).